Kudu.Core/Jobs/ContinuousJobsManager.cs

```diff
@@ -57,6 +62,39 @@ public void EnableJob(string jobName)
             continuousJobRunner.EnableJob();
         }
 
+        public async Task<HttpResponseMessage> RequestPassthrough(string jobName, string path, HttpRequestMessage request)
```

Comment:
As far as I can tell, the ContinuousJobRunner is the one that manages all the runtime needs for the continuous webjobs.